.github/workflows/build_etiss.yml

```diff
@@ -0,0 +1,133 @@
+name: Build ETISS
```

Comment:
The workflow name and filename should be a bit more precise:
- We do not only build etiss. The workflow runs m2isar, patches etiss and then builds it
- This workflow is currently S4EMac specific. This should probably be indicated somewhere
- Ideally we could extend this to support other archs than `RV32IMCXS4EMAC`. This should be feasible by moving the `env:` contents into a CI matrix and adding conditionals tto the steps below. I think we should discuss this after the PR is merged as it would require more testing...